### PR TITLE
Urfave cli v2

### PR DESCRIPTION
**Pull request by ffrank**

### CODE REVIEW

File: lang/gapi.go [hunk 1]
```diff
@@ -67,20 +67,20 @@ type GAPI struct {
...
 	result := []cli.Flag{}
 	modulePath := cli.StringFlag{
-		Name:   flagModulePath,
-		Value:  "", // empty by default
-		Usage:  "choose the modules path (absolute)",
-		EnvVar: "MGMT_MODULE_PATH",
+		Name:    flagModulePath,
+		Value:   "", // empty by default
+		Usage:   "choose the modules path (absolute)",
+		EnvVars: []string("MGMT_MODULE_PATH"),
```
Comment (purpleidea) [inline lang/gapi.go]: I bet you meant:

```
[]string{"MGMT_MODULE_PATH"},
```

**Comment by purpleidea** [comment]
You might also need to update something here: https://github.com/purpleidea/mgmt/blob/master/gapi/gapi.go#L104

File: lib/cli.go [hunk 2]
```diff
@@ -41,207 +41,207 @@ func CLI(program, version string, flags Flags) error {
 	}
 
 	// All of these flags can be accessed in your GAPI implementation with
-	// the `c.Parent().Type` and `c.Parent().IsSet` functions. Their own
+	// the `c.Lineage()[1].Type` and `c.Lineage()[1].IsSet` functions. Their own
```
Comment (purpleidea) [inline lib/cli.go]: It's especially dope when you also port the comments. I do this too. Not being sarcastic. :thumbsup:

File: lang/gapi.go [hunk 3]
```diff
@@ -110,11 +110,11 @@ func (obj *GAPI) CliFlags(command string) []cli.Flag {
 	case gapi.CommandDeploy:
 		flags := []cli.Flag{
 			// TODO: removed (temporarily?)
-			//cli.BoolFlag{
+			//*cli.BoolFlag{
 			//	Name:  "stdin",
 			//	Usage: "use passthrough stdin",
 			//},
-			modulePath,
+			&modulePath,
```
Comment (purpleidea) [inline lang/gapi.go]: actually I think you want the & up higher in the modulePath definition.

eg:  	`modulePath := cli.StringFlag{` becomes:

` 	modulePath := &cli.StringFlag{`

Comment (ffrank) [reply lang/gapi.go]: Ah. Yes, this is silly. It was one of the first places I touched and had not yet grokked this.

**Comment by purpleidea** [comment]
LGTM =D ... Waiting for travis.

PS: Did you smoke test to see if the nested GAPI stuff works?

**Comment by purpleidea** [comment]
Merged, thanks!

**Pull request merged**